pages/models.py

```diff
@@ -615,6 +620,29 @@ class Adopters(Page):
     parent_page_types = ['pages.HomePage']
 
 
+class Organizations(models.Model):
```

Comment:
I'd like to keep this name for the time being.  'Organizations' is the name that the salesforce team uses to reference this data (even though it's queried from a table call 'Accounts').  Maybe refactor this in an update?